packages/lesswrong/server/notificationTypesServer.tsx

```diff
@@ -331,27 +330,6 @@ export const NewPublishedDialogueMessageNotification = serverRegisterNotificatio
   },
 });
 
-export const NewDialogueMatchNotification = serverRegisterNotificationType({
```

Comment:
I think we probably shouldn't remove the notification types, since anyone with existing notifications of those types will be relying on this code to correctly render them

Reply:
This server side code I think should be safet to remove though, right? As it's for email notifs that should have gone out by now

Reply:
Hm... seems right, but would double-check with @jimrandomh

Reply:
Removing the server-side stuff for the notification without removing the client-side stuff may theoretically, work, but it's the sort of thing that could cause weird issues (eg, if a unit test checks that they match). I think the right thing to do here might be to have a migration delete old notifications of the deleted notification type?

That said, I think this is unlikely to matter much either way, and it's fine to pretty much do whatever. I've seen missing-notification-type errors happen in development (when a notification of a new type is created in the dev DB in a different branch than the one that's currently running), and while the notification that has an unrecognized type gets replaced with an error message, it doesn't break anything beyond that one notification. Users are unlikely to see that, since anyone who was active enough to have gotten notifications of these types will probably have gotten more notifications since then, pushing it out of the most-recent list.